pkg/machine/qemu/machine.go

```diff
@@ -1229,6 +1261,10 @@ func (v *MachineVM) startHostNetworking() (string, apiForwardingState, error) {
 		return "", noForwarding, err
 	}
 	binary, err := cfg.FindHelperBinary(machine.ForwarderBinaryName, false)
+	// workaround for lookup on Windows
```

Comment:
Will not be needed, when https://github.com/containers/common/issues/1123 is fixed.